### PR TITLE
docs: Add some common cases that have invalid checksum warning

### PR DESCRIPTION
**Pull request by thangngoc89**

### CODE REVIEW

File: docs/content/docs/faq.md [hunk 1]
```diff
@@ -14,3 +14,29 @@ Warning: React attempted to reuse markup in a container but the checksum was inv
...
+### Date and time
+
+Due to the different timezone between build server and your visitors, generated
+date and can be different. You should set timezone when generating date.
+
+Example with `moment.js`
+
+```diff
+-- moment(data).format("DD MMM YYYY")
+++ moment(date).utc().format("DD MMM YYYY")
```
Comment (MoOx) [inline docs/content/docs/faq.md]: If you can also provide a vanilla js example that will be nice :p

Comment (thangngoc89) [inline docs/content/docs/faq.md]: I'm researching it. Never manipulate date and time without moment.js

**Comment by thangngoc89** [comment]
@MoOx good enough ? http://jsbin.com/wegigotejo/edit?js,console

**Comment by thangngoc89** [comment]
```diff
const publicDate = new Date('March 1, 2010 00:00:00')
++ const currentDate = new Date()
++ const timezoneDiff = currentDate.getTimezoneOffset() / 60
++ publicDate.setHours(publicDate.getHours() + timezoneDiff)

console.log(publicDate.toDateString())
```

**Comment by MoOx** [comment]
Why don't you use utc() method? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC
You have all date.(s|g)etUTC* methods available.

**Comment by thangngoc89** [comment]
Oops. I missed that 

```
publicDate.toUTCString()
```

**Pull request merged**